### PR TITLE
some fixes to the readme install instructions

### PR DESCRIPTION
**Pull request by whyrusleeping**
cc @RichardLitt and @jbenet 

License: MIT
Signed-off-by: Jeromy <jeromyj@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,19 +82,12 @@ $ cd $GOPATH/src/github.com/ipfs/go-ipfs
 $ make install
 ```
 
-Tip:
-
-If the `make install` hangs while fetching dependencies and if you are
-running a daemon locally, ensure that it is on at least version 0.4.0-dev. If
-you don't have a build that is new enough, run the `make install` with your
-daemon turned off and the deps will be fetched through the ipfs.io gateways.
```
Comment (jbenet) [inline README.md]: Is this tip no longer needed? it's not findable, the install process should warn about it or something.

Comment (whyrusleeping) [inline README.md]: it won't be needed after #2394

File: README.md [hunk 2]
```diff
@@ -87,19 +82,12 @@ $ cd $GOPATH/src/github.com/ipfs/go-ipfs
...
-
 NOTES:
 
 * `git` is required in order for `go get` to fetch
 all dependencies.
 * Package managers often contain out-of-date `golang` packages.
-  Compilation from source is recommended.
+  ensure that `go version` reports at least 1.5.2.
```
Comment (RichardLitt) [inline README.md]: Ensure

Comment (RichardLitt) [inline README.md]: Can we add a line about how to ensure that, here?

Comment (whyrusleeping) [inline README.md]: yeah, i can say "see above on how to install go", will that work?

Comment (RichardLitt) [inline README.md]: Is there a command to update golang deps?

Comment (whyrusleeping) [inline README.md]: what do you mean by golang deps?

Comment (RichardLitt) [inline README.md]: Nevermind. Saying "See above on how to install go" works.

File: README.md [hunk 3]
```diff
@@ -87,19 +82,12 @@ $ cd $GOPATH/src/github.com/ipfs/go-ipfs
...
-you don't have a build that is new enough, run the `make install` with your
-daemon turned off and the deps will be fetched through the ipfs.io gateways.
-
 NOTES:
 
 * `git` is required in order for `go get` to fetch
 all dependencies.
 * Package managers often contain out-of-date `golang` packages.
-  Compilation from source is recommended.
+  ensure that `go version` reports at least 1.5.2. See above for how to install go.
```
Comment (RichardLitt) [inline README.md]: Capital E.

**Pull request merged**